tests/test_handlers.py

```diff
@@ -549,8 +549,8 @@ def test_am1_differences():
         if isinstance(ccc, nonbonded.AM1CCCHandler):
             break
 
-    suppl = Chem.SDMolSupplier("tests/data/ligands_40.sdf", removeHs=False)
-    smi = "[H]c1c(OP(=S)(OC([H])([H])C([H])([H])[H])OC([H])([H])C([H])([H])[H])nc(C([H])(C([H])([H])[H])C([H])([H])[H])nc1C([H])([H])[H]"
+    # suppl = Chem.SDMolSupplier("tests/data/ligands_40.sdf", removeHs=False)
+    # smi = "[H]c1c(OP(=S)(OC([H])([H])C([H])([H])[H])OC([H])([H])C([H])([H])[H])nc(C([H])(C([H])([H])[H])C([H])([H])[H])nc1C([H])([H])[H]"
```

Comment:
Did you intend to remove this? If not, a comment explaining the relevance would be useful

Reply:
Ah, I see you mentioned this in the message for ebd390f. Any reason to keep it around if unused?

Reply:
`suppl` looks safe to remove but I don't know what that `smi` is intended to exercise (maybe intended to exercise missing patterns for phosphorus in AM1CCCHandler?)

Reply:
I think `P` is not supported by the CCC handler, and I was testing this against AM1BCC to check for consistency. Wasn't sure what to do this but I probably should've indicated this.